### PR TITLE
Empty checkbox bypass

### PR DESCRIPTION
**Pull request by bradgarropy**
In order to address #345 I added a check for an empty value in the `checkbox` bypass.

```javascript
    if (v === "") {
      return [];
    }
```

I think this solves the issue, but I haven't been able to test it from the top level. Is there another test file that I'm missing?

/cc @amwmedia @crutchcorn

### CODE REVIEW

**Comment by crutchcorn** [comment]
Oh, nice, this was an easy fix! :D 

I would love to test this from an e2e level before we merge. Would you be comfortable making that change?

Namely, I think this file would be a good place to put it:

https://github.com/plopjs/plop/blob/main/packages/plop/tests/input-processing.spec.js

Happy to pair-code to make this happen :) Let me know how I can help

**Comment by bradgarropy** [comment]
Just pushed some additional tests for checkbox bypassing!

I feel like it would also be helpful to confirm that the checkbox value is actually an empty array `[]` when bypassed with an empty string `""`.

What do you think @crutchcorn? Is there a good place for that test?

**Review by crutchcorn — APPROVED** [review]
Love the type tests - great work!

Waiting for CI then will merge

**Comment by DawidWraga** [comment]
having issues inputting "" inside cli eg gen hook "useHook" "" as shell is automatically removing empty strings

**Pull request merged**
